### PR TITLE
Add raw_result option to appdynamics query_logs

### LINKED ISSUE #127: Appdynamics: allow returning raw result from query_logs
This is useful when query result include aggregation results. Right now returning only result ``hits`` which won't be sufficient in that case.

### PR DESCRIPTION
**Pull request by mohabusama**
Useful when caller interested in aggregation results and not only hits.

Resolve #127

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/appdynamics.py [hunk 1]
```diff
@@ -194,7 +192,7 @@ def healthrule_violations(self, application, time_range_type=BEFORE_NOW, duratio
             logger.exception('AppDynamics request failed')
             raise
 
-    def query_logs(self, q='', body=None, size=100, source_type=SOURCE_TYPE_APPLICATION_LOG, duration_in_mins=5):
+    def query_logs(self, q='', body=None, size=100, duration_in_mins=5, raw_result=False):
```
Comment (Jan-M) [inline zmon_worker_monitor/builtins/plugins/appdynamics.py]: Where did the source type default go to?

Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/appdynamics.py]: It is no longer fixed, changed to be unique per application. I believe depending on custom fields in filtering is very specific and could lead to issues. User can still filter using ``q`` param if required.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**